frontend/src/app/commonComponents/Header.jsx

```diff
@@ -193,7 +193,7 @@ const Header = ({ organizationId }) => {
               <div
                 ref={staffDefailsRef}
                 aria-label="Primary navigation"
-                className={classNames("usa-nav", "prime-staff-infobox", {
```

Comment:
I am wondering whether this was mis-marked as a `usa-nav` or not. The `aria-label` says it is for primary navigation and that seems wrong too?  Seems like a label of "User navigation" or something would be closer. Also I think this should be a `<button>` since it initiates an action.

Reply:
Oh i see, there's an Anchor below.

Reply:
Yeah, I'm adding some more changes to this pull request right now.

The nav region is very opinionated and I'm having to override a bunch of default styles/behaviors to get the desired results.